extensions/redis-client/runtime/src/main/java/io/quarkus/redis/runtime/client/config/RedisClientConfig.java

```diff
@@ -56,7 +56,7 @@ public interface RedisClientConfig {
     /**
      * The master name (only considered in HA mode).
      */
-    @ConfigDocDefault("my-master")
+    @ConfigDocDefault("mymaster")
```

Comment:
This is what Vert.x Redis client defaults to, and also what the Redis sentinel documentation uses in the examples.